vcpkg/commands/test-features.md

```diff
@@ -0,0 +1,113 @@
+---
+title: vcpkg x-test-features
+description: Command line reference for the vcpkg test-features command. Builds all features of a package.
+ms.date: 03/07/2025
+---
+
+# vcpkg test-features
+
+## Synopsis
+
+```console
+vcpkg x-test-features [options] <package>...
+```
+
+## Description
+
+Tests all features of a package by installing them in manifest mode. This command is useful for testing that all features of a package are working correctly.
+The following tests are performed:
+- The core feature gets installed with default featured disabled
+- Every feature gets installed separately with default features disabled
+- All features combined gets installed
+
+Feature which are unsupported or marked as cascade in --ci-feature-baseline are skipped.
+
+## Example
+
+```console
+$ $ ./vcpkg x-test-features curl
+Computing 23 install plans...
+Computing all ABI hashes...
+Detecting compiler hash for triplet x64-linux...
+Compiler found: /usr/bin/c++
+Checking the binary cache...
+Feature Test [1/23] curl[core]:x64-linux
+Feature Test [2/23] curl[core,non-http]:x64-linux
+Feature Test [3/23] curl[core,tool]:x64-linux
+...
+```
+
+## Options
+
+All vcpkg commands support a set of [common options](common-options.md).
+
+### `--ci-feature-baseline`
+
+Path to the `ci.feature.baseline.txt` file. Used to skip ports and detect regressions.
+
+Each line in this file is an assignment of a port spec to a state. `#`s indicate comments, and
```

Comment:
I think in this context it's not clear what a port spec is.

Reply:
I think it was clear given subsequent examples but I added a few more just in case.